### PR TITLE
SerCx: Add documentation for device interface creation

### PR DESCRIPTION
**Pull request by alscarpan**
This change adds documentation for a SerCx/SerCx2 feature that allows for device interface publication from the Cx.

Also updates ToC, and modernizes overview page from a HTML table to markdown.

### CODE REVIEW

File: windows-driver-docs-pr/serports/accessing-a-device-on-a-sercx2-managed-serial-port.md [hunk 1]
```diff
@@ -11,37 +11,22 @@ SerCx2 and a serial controller driver jointly manage a serial port to which a pe
...
+
+- [SerCx2 Handling of Read and Write Requests](sercx2-handling-of-read-and-write-requests.md)
+
+    A peripheral driver sends write ([**IRP_MJ_WRITE**](/previous-versions/ff546904(v=vs.85))) and read ([**IRP_MJ_READ**](/previous-versions/ff546883(v=vs.85))) requests to a port on a serial controller to transfer data to and from a peripheral device that is connected to the port. The way in which SerCx2 handles these requests is well-defined, even when the requests time out or are canceled.
+
+- [Reading Data from a SerCx2-Managed Serial Port](reading-data-from-a-sercx2-managed-serial-port.md)
+
+    A serial controller (or UART) typically includes a receive FIFO. This FIFO provides hardware-controlled buffering of data received from the peripheral device that is connected to the serial port. To read data from the receive FIFO, the peripheral driver for this device sends read ([**IRP_MJ_READ**](/previous-versions/ff546883(v=vs.85))) requests to the serial port.
```
Comment (mhopkins-msft) [inline windows-driver-docs-pr/serports/accessing-a-device-on-a-sercx2-managed-serial-port.md:28]: Can the link to IRP_MJ_READ point to this topic instead? **[IRP_MJ_READ](/windows-hardware/drivers/kernel/irp-mj-read)**

File: windows-driver-docs-pr/serports/accessing-a-device-on-a-sercx2-managed-serial-port.md [hunk 2]
```diff
@@ -11,37 +11,22 @@ SerCx2 and a serial controller driver jointly manage a serial port to which a pe
...
 
+- [Opening a SerCx2-Managed Serial Port](opening-a-sercx2-managed-serial-port.md)
+
+    If your peripheral driver controls a device on a serial port that is jointly managed by SerCx2 and a serial controller driver, your driver can open a logical connection to this port and then send I/O requests to the device through the port.
+
+- [SerCx2 Handling of Read and Write Requests](sercx2-handling-of-read-and-write-requests.md)
+
+    A peripheral driver sends write ([**IRP_MJ_WRITE**](/previous-versions/ff546904(v=vs.85))) and read ([**IRP_MJ_READ**](/previous-versions/ff546883(v=vs.85))) requests to a port on a serial controller to transfer data to and from a peripheral device that is connected to the port. The way in which SerCx2 handles these requests is well-defined, even when the requests time out or are canceled.
```
Comment (mhopkins-msft) [inline windows-driver-docs-pr/serports/accessing-a-device-on-a-sercx2-managed-serial-port.md:24]: Can the link to IRP_MJ_WRITE point to this topic instead? 
https://learn.microsoft.com/en-us/windows-hardware/drivers/kernel/irp-mj-write

**Review by mhopkins-msft — CHANGES_REQUESTED** [review]
Looks good, just a couple minor link changes.

**Comment by alscarpan** [comment]
> Looks good, just a couple minor link changes.

Thanks Mark, have made the link updates.

**Review by mhopkins-msft — APPROVED** [review]

**Comment by mhopkins-msft** [comment]
@alscarpan, let me know when you're ready to publish and I'll merge it.

**Pull request merged**